### PR TITLE
langsmith 0.1.129 :snowflake:

### PR DESCRIPTION
**Pull request by tombenes**
langsmith 0.1.129 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5823]
- dev_url:        https://github.com/langchain-ai/langsmith-sdk/tree/v0.1.129
- conda_forge:    https://github.com/conda-forge/langsmith-feedstock
- pypi:           https://pypi.org/project/langsmith/0.1.129
- pypi inspector: https://inspector.pypi.io/project/langsmith/0.1.129

### Explanation of changes:

- new version number

[PKG-5823]: https://anaconda.atlassian.net/browse/PKG-5823?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,6 +22,7 @@ requirements:
     - poetry-core
     - pip
   run:
+    - httpx >=0.23.0,<1
     - python
     - pydantic >=1.0.0,<3.0.0
```
Comment (msentissi) [inline recipe/meta.yaml]: Given that the `.toml` file specifies 2 compatible but different constraints depending on the exact python version, I would err on the side of caution and use the intersection of constraints. What do you think ?
```suggestion
    - pydantic >=2.7.4,<3.0.0
```

Comment (tombenes) [reply recipe/meta.yaml]: I went with selectors to reflect source 1:1

**Comment by skupr-anaconda** [comment]
@Cansisti A title should be renamed `langsmith 0.1.129` with corresponding links in the PR's initial comment, right?

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (msentissi) [reply recipe/meta.yaml]: I'm not sure that selectors are precise enough to match the source (3.12.**4** cutoff) ?

**Review by msentissi — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: The original issue was https://github.com/pydantic/pydantic/issues/9607 and I think it's fine to not include all py312 for pydantic 1

Comment (msentissi) [reply recipe/meta.yaml]: Looking at it again, I see that using the specifiers is stricter than the exact cutoff used in the source. I agree this should work.

**Comment by tombenes** [comment]
> @Cansisti A title should be renamed `langsmith 0.1.129` with corresponding links in the PR's initial comment, right?

Updated

**Pull request merged**